docs/source/posts/2022/2022-07-25-week-6-shivam.rst

```diff
@@ -0,0 +1,42 @@
+Week 6 - Extracting the animation data
+======================================
+.. post:: July 25 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- This week, it was all about reading docs and extracting the animation data from buffers.
+
+- Currently, the glTF class can extract simple node transformation and morphing data, As they are stored in the ``Animations`` of the glTF file.
+
+- Skinning (Skeletal Animation) data is stored in ``Nodes`` inside the skin parameter. We shall be able to load that before our next meeting on Wednesday.
+
+- Created a `tutorial <https://github.com/xtanion/fury/blob/gltf-anim-merge-kf/docs/tutorials/01_introductory/viz_simple_gltf_animation.py>`_ using keyframe animations (`#626`_.) and adding multiple ``timelines`` into a main ``timeline`` as suggested by Mohamed.
+
+  .. raw:: html
+
+        <iframe id="player" type="text/html"   width="550" height="400" src="https://user-images.githubusercontent.com/74976752/180841262-7356204e-2097-4ea4-ab71-39f04a1e7a07.mp4" frameborder="0"></iframe>
+
+
+
+What is coming up next week?
+----------------------------
+
+As of now, we've decided the following:
+
+- Create a custom Interpolator.
+
+Other tasks are yet to be decided.
```

Comment:
please, remove this, and propose tasks that you know you will do. 

This sentence makes people believe that you are done and just waiting without any task idea. Same remark for @m-agour and @ganimtron-10. Please, avoid this sentence all and explain what you REALLY plan to do. Our meetings are just to update the plan.